.github/workflows/build.yml

```
@@ -12,7 +12,7 @@ jobs:
     name: Ruby ${{ matrix.version }}
     strategy:
       matrix:
-        version: [2.5, 2.6, 2.7]
+        version: [2.5, 2.6, 2.7, '3.0']
```

Let's better remove 2.5 to just keep supported versions. Also, does 3.0 require the quotes? 🤔 

```suggestion
        version: [2.6, 2.7, 3.0]
```

---

Removed 2.5. Quotes for `3.0` are required due to a bug in Github Actions runner: https://github.com/actions/runner/issues/849